lib/models/project.js

```diff
@@ -338,7 +339,8 @@ Project.prototype.supportedInternalAddonPaths = function() {
     path.join(internalMiddlewarePath, 'tests-server'),
     path.join(internalMiddlewarePath, 'history-support'),
     path.join(internalMiddlewarePath, 'serve-files'),
-    path.join(internalMiddlewarePath, 'proxy-server')
+    path.join(internalMiddlewarePath, 'proxy-server'),
+    path.join(this.emberCliNodeModulesPath, 'ember-cli-legacy-blueprints')
```

Comment:
in what scenarios does it get addon automatically?

* in all cases?
* in cases where the ember-core addon is missing